docs/content/reference/operating_systems_support.md

```diff
@@ -7,9 +7,14 @@ weight: 10
 tags: ["setup", "reference", "windows", "linux"]
 ---
 
-The Infection Monkey project supports many popular OSes (but we are always interested in supporting more).
+The Infection Monkey project supports many popular operating systems.
 
-The Infection Monkey Agent has been tested to run on the following operating systems (on the x86_64 architecture):
+The Agent has a wider support than the Monkey Island (control server) to make sure that users can
+test even their legacy systems' security.
+
+{{% notice info %}}
+Infection Monkey is not supported on ARM-based or 32-bit systems.
```

Comment:
How about PowerPC or S390X? Sparc?

I think in this case it would be better to list only those supported architectures than the unsupported ones.

Reply:
I don't think there's anyone running S390X and expecting it to work :D But I can change if you insist